### PR TITLE
`azurerm_bot_service_azure_bot` - support for `local_authentication_enabled`

### PR DESCRIPTION
**Pull request by neil-yechenwei**
image

### CODE REVIEW

File: internal/services/bot/bot_service_resource_base.go [hunk 1]
```diff
@@ -107,6 +107,12 @@ func (br botBaseResource) arguments(fields map[string]*pluginsdk.Schema) map[str
 			}, false),
 		},
 
+		"local_auth_enabled": {
```
Comment (tombuildsstuff) [inline internal/services/bot/bot_service_resource_base.go:110]: this should be `local_authentication_enabled` to match other cases in the provider - can we update this?

Comment (neil-yechenwei) [reply internal/services/bot/bot_service_resource_base.go]: Updated

**Comment by neil-yechenwei** [comment]
@tombuildsstuff , thanks for the comment. I updated PR. Please take another look. Thanks.

**Review by stephybun — APPROVED** [review]
Thanks @neil-yechenwei LGTM 👍

**Pull request merged**